tdrs-backend/docs/api/report.md

```diff
@@ -0,0 +1,64 @@
+# Report metadata
+
+Accepts s3 slugs and maintains every prior version of the file
+
+
+**Request**:
+
+`POST` `/v1/reports/`
+
+Parameters:
+```json
+{
+
+            "original_filename":"filename", //name of the file before it was uploaded to s3.
+            "slug":"hdslajhfdaksdjflajlsdfa",
+            "extension":"txt",//optional
+            "user":"lnsdfkldlkajdfa",// uuid
```

Comment:
re: slug and user--could we format these to look more like a user or slug string/identifier?

Reply:
@ADPennington I have updated this